tests/pod.py

```diff
@@ -67,3 +67,19 @@ def delete_pod(pod_name, name_space):
 
     wait_to_pod_to_be_deleted(pod_name, name_space)
     return resp
+
+
+def check_for_restarted_pods(namespace, specific_deployment_name=''):
+    if specific_deployment_name:
+        pods = client.CoreV1Api().list_namespaced_pod(namespace,
+                                                      label_selector=(
+                                                          "name={0}".format(specific_deployment_name.split('-')[0]))).items
+    else:
+        pods = client.CoreV1Api().list_namespaced_pod(namespace).items
```

Comment:
I don't really get how `pods` work for you if you only assign `pods` inside an ifelse block.